docusaurus/docs/learn/core-concepts/services/overview.mdx

```diff
@@ -143,3 +143,339 @@ This strategy drives costs in the same way as the `smartrouting` strategy. Howev
 ##### `random`
 This strategy does not change terminator weights. It does simple random selection across all terminators of the highest precedence. 
 
+## Practical Service Hosting
```

Comment:
What's the goal of this heading? "Practical Hosting"? I'd expect to see something like "Hosting Services", then go into  "Edge Router Hosted" and then to "Healthchecks" based on the PR name

Reply:
Changed to just `Service Hosting`